<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceTestDialogClass</class>
 <widget class="QDialog" name="DeviceTestDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Testing APO installation on devices...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTreeWidget" name="deviceTreeWidget">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
     </property>
     <column>
      <property name="text">
       <string>Connector</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Device</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Pre-mix</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Post-mix</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="statusOutputBox">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
